modules/OsmOsisManager.py

```diff
@@ -322,57 +284,48 @@ def run_diff(self, conf):
         self.logger.log("iteration=%d" % nb_iter)
 
         try:
-          cmd  = [conf.bin_osmosis]
-          cmd += ["--read-replication-interval", "workingDirectory=%s" % diff_path]
-          cmd += ["--simplify-change", "--write-xml-change", "file=%s" % xml_change]
-          cmd += ["-quiet"]
+          cmd  = [conf.bin_pyosmium_up_to_date]
+          cmd += ["-v"]
+          cmd += ["--format", "pbf"]
+          cmd += ["--outfile", tmp_pbf_file]
```

Comment:
Est-ce que tu peux rajouter un rm de tmp_pbf_file ?

J'ai remarqué que pyosmium-up-to-date plante sur le fichier de sortie existe déjà.

Reply:
Done